morebits.js

```diff
@@ -1762,7 +1762,7 @@ Morebits.wiki.api.prototype = {
 		var queryString = $.map(this.query, function(val, i) {
 			if (Array.isArray(val)) {
 				return encodeURIComponent(i) + '=' + val.map(encodeURIComponent).join('|');
-			} else if (val !== undefined) {
+			} else if (val !== undefined && val !== null) {
```

Comment:
So if we say `undefined`, `null`, and `false` are out, that becomes, what


```suggestion
			} else if (val !== undefined && val !== null && val !== false) {
```

It'd be nicer visually to do:

```suggestion
			} else if (val || val === '') {
```

But that would remove `0` and `NaN` — I can't imagine ever wanting to pass `NaN` but it's certainly possible to want `0` (number).